public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx

```diff
@@ -169,6 +170,11 @@ export const QueryWrapper = ({
     return (
       <Stack direction="row" alignItems="center" gap={1}>
         <SelectingDataSourceTooltip />
+        <AlertingRuleQueryExtentionPoint
+          query={Object.assign({}, query.model)}
+          ruleFormValues={getValues()}
+          extensionsToShow="queryless"
```

Comment:
might be interesting to allow multiple extensions to be shown, perhaps an array of extensions would be preferable?

Reply:
@gillesdemey Here is the extension point that is one way, and allows for more extension points in the `AlertingRuleQueryExtensionPoint` component. Currently it only allows for drilldown apps but if you want to add more etension points in the query editor, I added a TODO comment where I think it would fit nicely.

https://github.com/grafana/grafana/pull/108566